test.js

```diff
@@ -0,0 +1,78 @@
+require( 'sugar' );
+var assert = require( 'assert' );
+var Promise = require( 'bluebird' );
+var parseBig = require( './index' );
+
+var MAX_SAFE_INTEGER = Number.MAX_SAFE_INTEGER;
+var MIN_SAFE_INTEGER = Number.MIN_SAFE_INTEGER;
+
+describe( 'Parse Big', function () {
+    it( 'error', function ( done ) {
+        var badJsonString = '{"name":"somename}';
+
+        Promise.resolve( badJsonString )
```

Comment:
@kensaggy I think it's more readable and tidy, especially when handling errors that might occur during the test, but since it is a big dependency, and for the purposes of this test I'm removing it.